src/components/info-window.tsx

```diff
@@ -176,6 +177,47 @@ export const InfoWindow = (props: PropsWithChildren<InfoWindowProps>) => {
     const openOptions: google.maps.InfoWindowOpenOptions = {map};
     if (anchor) {
       openOptions.anchor = anchor;
+
+      // Only do the infowindow adjusting when dealing with an AdvancedMarker
+      if (isAdvancedMarker(anchor) && anchor.content instanceof Element) {
+        const {width: anchorWidth, height: anchorHeight} =
+          anchor.content.getBoundingClientRect() ?? {};
+
+        // This checks whether or not the anchor has custom content with our own
+        // div wrapper. If not, that means we have a regular AdvancedMarker without any children.
+        // In that case we do not want to adjust the infowindow since it is all handled correctly
+        // by the Google Maps API.
+        if (anchorWidth === 0 && anchorHeight === 0) {
+          // We can safely typecast here since we control that element and we know that
+          // it is a div
+          const anchorDomContent = anchor.content.firstChild as Element;
+          const anchorDomWidth =
+            anchorDomContent?.getBoundingClientRect().width;
+
+          const anchorComputedStyles = getComputedStyle(anchorDomContent);
+          const transformMatrix = new WebKitCSSMatrix(
```

Comment:
Not sure about the `Webkit` prefix, I think it's now called [`DOMMatrix`](https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix). 

If I understood this right, we need to compute the offset of the top/center of the marker-content to the current position of the marker. According to [the spec](https://drafts.csswg.org/cssom-view/#dom-element-getclientrects), getBoundingClientRect() already have all transforms applied, so we could maybe just take the values from there? Maybe the BCR of our 0x0 anchoring div could be used as reference to get to the actual offset.

Reply:
Good point :) Yes we can.